scribe/emitter.go

```diff
@@ -111,15 +111,35 @@ Entries:
 // name, whether or not it is a default process, the command, arguments, and
 // any process specific environment variables.
 func (e Emitter) LaunchProcesses(processes []packit.Process, processEnvs ...map[string]packit.Environment) {
+	launchProcesses := []packit.LaunchProcesses{}
+	for _, process := range processes {
+		launchProcesses = append(launchProcesses, process)
+	}
+	e.launch(launchProcesses, processEnvs...)
+}
+
+// LaunchDirectProcesses take a list of processes and a map of process specific
+// enivronment varables and prints out a formatted table including the type
+// name, whether or not it is a default process, the command, arguments, and
+// any process specific environment variables.
+func (e Emitter) LaunchDirectProcesses(processes []packit.DirectProcess, processEnvs ...map[string]packit.Environment) {
```

Comment:
Also related to the `process.ToDirectProcess()` conversation above, I'm wondering what the log emitter experience would be in the case where an indirect process is used with API 0.9.
Which log function would we expect a buildpack author to use in their buildpack? `emitter.LaunchProcesses`?